### ISSUE TITLE
Failed to open library database error on startup

### ISSUE THREAD

**Issue by dlardo** [JosefNemec/Playnite#780]
Issue started with the 4.5 update, but I'm still seeing it in 4.5.3.

Looking at the log, it appears to be a execution permission issue loading `LibraryExporter.ps1`. In previous versions I used to get a script execution policy permission error on startup that I ignored and everything still seemed to function. Unfortunately the machine I'm using is managed by a domain and changing the script execution policy is not an option for me.

```
2018-10-01 11:03:52.8582|ERROR:Failed to load script file C:\Users\dlardo\AppData\Local\Playnite\Extensions\LibraryExporter\LibraryExporter.ps1System.Management.Automation.CmdletInvocationException: Windows PowerShell updated your execution policy successfully, but the setting is overridden by a policy defined at a more specific scope.  Due to the override, your shell will retain its current effective execution policy of RemoteSigned. Type "Get-ExecutionPolicy -List" to view your execution policy settings. For more information please see "Get-Help Set-ExecutionPolicy". ---> System.Security.SecurityException: Security error.
   at System.Management.Automation.MshCommandRuntime.ThrowTerminatingError(ErrorRecord errorRecord)
   --- End of inner exception stack trace ---
   at System.Management.Automation.Runspaces.PipelineBase.Invoke(IEnumerable input)
   at System.Management.Automation.Runspaces.Pipeline.Invoke()
   at Playnite.Scripting.PowerShell.PowerShellRuntime..ctor()
   at Playnite.Scripting.PowerShell.PowerShellScript..ctor(String path)
   at Playnite.Scripting.PlayniteScript.FromFile(String path)
   at Playnite.Plugins.ExtensionFactory.LoadScripts(IPlayniteAPI injectingApi, List`1 ignoreList)
```

After the errors are presented, Playnite has no data and doesn't function. I've tried uninstalling and manually deleting the contents of %HOMEPATH%\AppData\Local\Playnite and %AppData%\Playnite\
 
[diag package.zip](https://github.com/JosefNemec/Playnite/files/2435114/diag.package.zip)

![failed to open db](https://user-images.githubusercontent.com/5000601/46307652-58b1fe80-c56c-11e8-9a69-be7e736aa195.PNG)

![database is not opened error 2](https://user-images.githubusercontent.com/5000601/46307653-58b1fe80-c56c-11e8-9d23-04a4241eac45.PNG)

**Comment by JosefNemec** [comment]
Can you try to disable `Library Exporter` extension in Settings -> Extensions menu, to see if it fixes the issue?

**Comment by dlardo** [comment]
Yes, that worked, thank you!

**Comment by JosefNemec** [comment]
Released in 4.55
